app/Renderers/Layouts/Campaign/Webhook.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Renderers\Layouts\Campaign;
+
+use App\Renderers\Layouts\Layout;
+
+class Webhook extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'action' => [
+                'key' => 'action',
+                'label' => 'campaigns.webhooks.fields.event',
```

Comment:
For our own sanity, please move all the labels to campaigns/webooks.php instead of inside of the already too big campaigns.php translation file